dataland-backend/src/main/kotlin/org/dataland/datalandbackend/frameworks/eutaxonomynonfinancials/model/capex/EutaxonomyNonFinancialsCapex.kt

```diff
@@ -0,0 +1,53 @@
+// <--WARNING--> THIS FILE IS AUTO-GENERATED BY THE FRAMEWORK-TOOLBOX AND WILL BE OVERWRITTEN
+package org.dataland.datalandbackend.frameworks.eutaxonomynonfinancials.model.capex
+
+import jakarta.validation.Valid
+import org.dataland.datalandbackend.frameworks.eutaxonomynonfinancials.custom.EuTaxonomyActivity
+import org.dataland.datalandbackend.frameworks.eutaxonomynonfinancials.custom.EuTaxonomyAlignedActivity
+import org.dataland.datalandbackend.frameworks.eutaxonomynonfinancials.model.capex.alignedShare.EutaxonomyNonFinancialsCapexAlignedShare
+import org.dataland.datalandbackend.frameworks.eutaxonomynonfinancials.model.capex.eligibleShare.EutaxonomyNonFinancialsCapexEligibleShare
+import org.dataland.datalandbackend.frameworks.eutaxonomynonfinancials.model.capex.nonAlignedShare.EutaxonomyNonFinancialsCapexNonAlignedShare
+import org.dataland.datalandbackend.frameworks.eutaxonomynonfinancials.model.capex.nonEligibleShare.EutaxonomyNonFinancialsCapexNonEligibleShare
+import org.dataland.datalandbackend.model.datapoints.CurrencyDataPoint
+import java.math.BigDecimal
+import kotlin.collections.MutableList
+
+/**
+ * The data-model for the Capex section
+ */
+data class EutaxonomyNonFinancialsCapex(
+    val totalAmount: CurrencyDataPoint? = null,
+
+    @field:Valid()
```

Comment:
Why is there the valid annotation but deeper there is no validator provided?

Reply:
@MarcTM01 hast du eine Idee? 

Ich war bei dem Validation-Zeug nicht involviert. Vllt kannst du das schneller beantworten.

Reply:
Valid annotations are created by default for any component-group as they might include things that need to be validated. I saw no advantage in creating additional logic to check if it is actually needed.